walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -22,59 +22,86 @@
 
 /** Finds potential meeting times. */
 public final class FindMeetingQuery {
+  public Collection<TimeRange> timesForEveryone;
+  public Collection<TimeRange> timesForConfirmed;
+  public int currentEndTime;
 
   /**
-   * Returns available TimeRanges for the {@code request} given {@code events}, a Collection of
-   * all events so far. 
+   * Returns available TimeRanges for the {@code request} given {@code events}. 
    */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    Collection<TimeRange> times = new ArrayList<>();
-    Collection<String> attendees = request.getAttendees();
-    
+    timesForEveryone = new ArrayList<>();
+    timesForConfirmed = new ArrayList<>();
```

Comment:
I think it made more sense when these used to be local variables and not class members. Your method is doing a lot. The JavaDoc claims it returns a TimeRange, but then it also sets all this internal state. Maybe the way it is implemented needs to be re-thought or this method needs to be split into multiple methods.